playwright/test/whats-on-eventsByMonth.test.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from '@playwright/test';
+import { whatsOnUrl } from './helpers/urls';
+import { gotoWithoutCache } from './helpers/contexts';
+
+test.describe('EventsByMonth with JavaScript disabled', () => {
+  test.use({ javaScriptEnabled: false }); // Turn off JS for tests in this block
+
+  test(`the what's on page displays all events`, async ({ page }) => {
```

Comment:
absolute nitpick, could we use single quotes to align the standard? (and for the one below)